### ISSUE TITLE
Unable to connect to D365 on-premise

### ISSUE THREAD

**Issue by tommyboisvert** [MscrmTools/MscrmTools.Xrm.Connection#71]
Since many weeks, I'm not able to connect to an on-premise Dynamics 365 environment. It was always working well before, but since many weeks, it's impossible to connect. Other people in my team are able to connect, using XRM Toolbox. It's not a login issue, as I tried to logon to CRM directly and it works. But when I use XRMToolbox, it fails all the time. I attached the log file. I also deleted XRMToolbox and I reinstalled it, it didn't help. I just updated to the latest version, it doesn't help. Just by looking at the error log. I have no idea what is the real issue. If you can help, that would be appreciated.

Thanks,

Tommy

[XrmToolBox_Connection-2018-08-02.log](https://github.com/MscrmTools/MscrmTools.Xrm.Connection/files/2254146/XrmToolBox_Connection-2018-08-02.log)

**Comment by MscrmTools** [comment]
Are you able to connect to the same organization using one of the official MS tool like plugin registration tool?

**Comment by MscrmTools** [comment]
Can you also ensure you are resetting XrmToolBox correctly? 
https://www.xrmtoolbox.com/documentation/for-users/reset-xrmtoolbox-installation/

**Comment by tommyboisvert** [comment]
Thanks for the quick follow-up. I tried with Plugin Registration Tools and it didn't work. I tried with a different user/pwd and it worked. So, I did create a new connection in XRMToolbox, using the other user/pwd, and it worked. I can connect to CRM so I know my user/pwd are good and I have access to CRM, but it looks like someone did something to my profile, because the issue is clearly related to my account. In CRM, my user has System Administrator security role, with a read-write / professional CAL. Everything seems ok but I can't connect using XRMToolbox or Plugin Registration Tool. Anyway, I'll use the other credentials so at least I can use XRMToolbox. Thanks!
